setup.py

```diff
@@ -28,6 +28,7 @@
     "sphinx_tabs",
     "pyppeteer",
     "beautifulsoup4",
+    "cookiecutter",
```

Comment:
@TomasBeuzen Should this dependency be added to `install_requires` rather than `test_reqs`.

Reply:
Good thinking -- the message is useful if `cookiecutter` can't be imported. I'm happy with the helpful message route here - makes sense.